airflow/Dockerfile

```diff
@@ -95,7 +94,12 @@ RUN chmod a+x /entrypoint.sh && \
     chmod a+x /run-airflow.sh && \
     chmod +x /usr/bin/tini
 
-COPY --from=statsd-exporter --chown=stackable:stackable /bin/statsd_exporter /stackable/statsd_exporter
+# Choosing correct binary with respect to architecture
```

Comment:
I could. The binary above Tini, line 90 is pulled the same way from the same repository for the same reason. Is that a part for the builder then too?